resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/spark-base/Dockerfile

```diff
@@ -34,6 +35,6 @@ COPY dockerfiles/spark-base/entrypoint.sh /opt/
 
 ENV SPARK_HOME /opt/spark
 
-WORKDIR /opt/spark
+WORKDIR /opt/spark/work-dir
```

Comment:
in a sense this is a backcompat break for people baking their own images that expected `/opt/spark` to be the cwd.

trading off this break for the ability to make a file named `jars` that conflicts with the folder with the same name -- is that the right tradeoff?

this won't affect any of the application images we make I think, so maybe not bad in general